lib/galaxy/jobs/__init__.py

```diff
@@ -1597,6 +1597,7 @@ def setup_external_metadata( self, exec_dir=None, tmp_dir=None,
                                                                       config_file=config_file,
                                                                       datatypes_config=datatypes_config,
                                                                       job_metadata=os.path.join( self.working_directory, TOOL_PROVIDED_JOB_METADATA_FILE ),
+                                                                      max_metadata_value_size = self.app.config.max_metadata_value_size,
```

Comment:
Yep, fixing.  I had relaxed my local settings before we implemented all this, need to update it :)